app.js

```diff
@@ -1,7 +1,20 @@
 const express = require('express');
 const app = express();
-const {getTopics} = require('./controllers/controller')
+const {getTopics, getArticles} = require('./controllers/controller')
 
 app.get('/api/topics', getTopics)
+app.get('/api/articles/:article_id', getArticles)
+
+app.use((err, req, res, next) => {
+    if(err.status && err.msg) {
+        res.status(err.status).send({msg: err.msg})
+    } else next(err)
+})
+
+app.use((err, req, res, next) => {
+    if(err.code === '22P02') {
+        res.status(400).send({msg: 'Invalid input'})
+    }
```

Comment:
This is missing an else to move onto the final error handler which may be missing, for a 500 server error.